.drone.yml

```diff
@@ -5632,3 +5632,154 @@ depends_on:
   - corr/indexer-tzkt-octez-009-florence
   - corr/indexer-tzkt-tezedge-009-florence
   - perf/wrk-tests-009-florence
+
+---
+kind: pipeline
+name: perf/memory-allocation-bootstrap-009-florence
```

Comment:
@akoptelov 
please:
- move it before `deployment/publish-docker-image-develop` pipeline
- add it to those docker pipelines `depends_on`